features/afsluiten.feature

```diff
@@ -0,0 +1,73 @@
+#language: nl
+
+Functionaliteit: Zaakdossiers moeten na afsluiten duurzaam toegankelijk gehouden worden. Om dit te mogelijk te maken moet aan een aantal voorwaarden worden voldaan.
+  Als een informatiebeheerder
+  wil ik dat zaakdossiers voor het afsluiten van de zaak beschikken over een minimale set kenmerken
+  zodat ik deze dossiers goed kan beheren en eventueel op het juiste moment kan vernietigen
+
+Scenario: afsluiten zaak - zaak heeft geen resultaat
+  Gegeven dat een medewerker de zaak met uuid "095be615-a8ad-4c33-8e9c-c7612fbf6c9f" wil afsluiten door het aanmaken van de eindstatus.
```

Comment:
We moeten in algemene zin heel voorzichtig zijn met het gebruik van bestaande ids al snap ik de keuze hier wel. Als we dit namelijk op een andere omgeving doen (laten we zeggen lokaal) dan bestaat deze zaak helemaal niet.

Hiervoor moet er denk ik een fixture komen die een zaak aanmaakt met bepaalde waarde. Gaarne dit even met @MatthijsBekendam en @michielverhoef bespreken hoe we zorgen dat we de juiste data hebben die we ook in de tests kunnen gebruiken.

Reply:
Aangezien het uuid hier overal wordt gebruikt zou je hier voor maar 1 zaak hoeven aan te maken. Als we die als json kunnen krijgen is het eenvoudig om tijdens de start een zaak aan te maken en daarna weer te verwijderen. Zie daarvoor:

`environment.py` waar er nu 1 fixture is en:

https://behave.readthedocs.io/en/stable/fixtures.html